### PR TITLE
[SDK-5070] Add Feature Flag expanded properties for Wrapper SDKs

### PR DESCRIPTION
**Pull request by vanessaland**
### Summary

My PR adds code examples for the Wrapper SDKs (React Native, Cordova, Unity, Flutter, Roku) for accessing the expanded Feature Flag properties. Also updated the comments to be consistent and easier to read.

### Related PRs, issues, or features (optional)

- [SDK-5070](https://jira.braze.com/browse/SDK-5070)

### Feature release date (optional)

- Next week (Week of July 22-26)

### Contributor checklist

- [x] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

File: _docs/_developer_guide/platform_wide/feature_flags/create.md [hunk 1]
```diff
@@ -283,141 +283,230 @@ If a feature flag is not enabled, or a property you reference does not exist, th
...
-// Number property as double
+
+// Returns the number property as a double
 var doubleProperty = featureFlag.getDoubleProperty("height");
+
+// Returns the Unix UTC millisecond timestamp property as a long
+var timestampProperty = featureFlag.getTimestampProperty("account_start");
+
+// Returns the image property as a String of the image URL
+var imageProperty = featureFlag.imageProperty("homepage_icon");
```
Comment (jerielng) [inline _docs/_developer_guide/platform_wide/feature_flags/create.md:432]: Should this be:
```suggestion
var imageProperty = featureFlag.getImageProperty("homepage_icon");
```

**Review by jerielng — APPROVED** [review]
LGTM - we should probably hold off on merging until each of the wrappers release.

**Review by hokstuff — APPROVED** [review]

File: _docs/_developer_guide/platform_wide/feature_flags/create.md [hunk 2]
```diff
@@ -283,141 +283,230 @@ If a feature flag is not enabled, or a property you reference does not exist, th
...
 {% tab Unity %}
 
 ```csharp
-// Feature flag instance
+// Returns the Feature Flag instance
 var featureFlag = Appboy.AppboyBinding.GetFeatureFlag("expanded_user_profile");
-// String properties
+
+// Returns the String property
 var stringProperty = featureFlag.getStringProperty("color");
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_wide/feature_flags/create.md:417]: Noting here – Based on [this comment](https://github.com/braze-inc/unity-sdk/pull/462#pullrequestreview-2210481713), we'll want to use Pascal Case for the already-released getter methods too (and deprecate the lowercase variants) in the next release

Once that is finalized and we figure out all the instances we need to change to Pascal Case, we can update this docs PR too 👍

**Comment by vanessaland** [comment]
Hi @internetisaiah , this PR is ready for Docs Team review!

We have been rolling out each of the Wrapper Releases that include these changes. This PR is still on hold until all are released, but I'll update you when it's ready to be merged. Thanks!

File: _docs/_developer_guide/platform_wide/feature_flags/create.md (hunk 2 above)
Comment (internetisaiah) [reply _docs/_developer_guide/platform_wide/feature_flags/create.md:417]: @hokstuff , was this completed? If so, would you mind resolving this convo? thanks!

**Review by internetisaiah — APPROVED** [review]
LGTM, just waiting on clarification on the currently open conversation. @vanessaland , just ping me here when this one's ready to be merged and released!

**Pull request merged**